src/Controls/src/Core/ScrollView/ScrollView.cs

```diff
@@ -171,10 +169,33 @@ public View Content
 			}
 		}
 
+		/// <inheritdoc cref="IInputTransparentContainerElement.CascadeInputTransparent"/>
+		public bool CascadeInputTransparent
```

Comment:
For all these properties, even though we moved them to `InputTransparentContainerElement` we still need to add the BindableProperty to this class as well

So for all uses of this (and other areas where this pattern was followed) we still need


```C#
/// <summary>Bindable property for <see cref="CascadeInputTransparent"/>.</summary>
		public static readonly BindableProperty CascadeInputTransparentProperty = InputTransparentContainerElement.CascadeInputTransparentProperty;
```